backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/starrating/BpkStarRating.kt

```diff
@@ -0,0 +1,151 @@
+/*
+ * Backpack for Android - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package net.skyscanner.backpack.compose.starrating
+
+import androidx.compose.foundation.layout.Row
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.semantics.contentDescription
+import androidx.compose.ui.semantics.semantics
+import kotlin.math.ceil
+import kotlin.math.floor
+import kotlin.math.round
+import net.skyscanner.backpack.compose.icon.BpkIcon
+import net.skyscanner.backpack.compose.icon.BpkIconSize
+import net.skyscanner.backpack.compose.theme.BpkTheme
+import net.skyscanner.backpack.compose.tokens.Star
+import net.skyscanner.backpack.compose.tokens.StarHalf
+import net.skyscanner.backpack.compose.tokens.StarOutline
+
+enum class BpkStarRatingSize {
+  Large,
+  Small,
+}
+
+@Composable
+fun BpkStarRating(
+  rating: Float,
+  contentDescription: String,
+  modifier: Modifier = Modifier,
+  rounding: RoundingType = RoundingType.Down,
+  size: BpkStarRatingSize = BpkStarRatingSize.Small,
+) {
+  BpkStarRating(
+    maxRating = 5,
+    rounding = rounding,
+    iconSize = when (size) {
+      BpkStarRatingSize.Large -> BpkIconSize.Large
+      BpkStarRatingSize.Small -> BpkIconSize.Small
+    },
+    modifier = modifier,
+    rating = rating,
+    contentDescription = contentDescription,
+  )
+}
+
+@Composable
+fun BpkHotelRating(
+  rating: Float,
```

Comment:
as the hotels version only supports full stars should this be an integer? otherwise it might look like it supports half stars